engine/edge.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Docker CE Edge releases
+description: Information and changelogs for Docker CE Edge releasers
+---
+
+## Overview
+
+Docker CE Edge releases are monthly releases cut straight from the `master`
+branch of the Docker repository. These releases allow you to try the latest
+features of Docker in non-production environments. They are effectively previews
+of the next Docker CE Stable release. Unless problems are found, these features
+will be available in the next releases of Docker CE and Docker EE.
+
+> **Warning**: Docker CE Edge releases do not receive bug fixes, backports, or
```

Comment:
AFAIK, this is not true; they are EOL'd after a month, but _can_ receive a patch release within that month